fmperf/utils/constants.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+REQUESTS_DIR = (lambda s: s + "/" if len(s) > 1 and not s.endswith("/") else s)(
+    os.environ.get("REQUESTS_DIR", "")
+)
```

Comment:
Do we really need this complex logic with the lamba function? If we just use `os.path.join` we shouldn't need it right?

Reply:
You're right, my brain was hallucinating at this point.